unetsocket/c/unet.c

```diff
@@ -151,6 +167,23 @@ unetsocket_t unetsocket_open(const char* hostname, int port) {
     fjage_subscribe_agent(usock->gw, agents[i]);
   }
   free(agents);
+  // check the parameter request class name
+  fjage_msg_t msg;
+  fjage_aid_t aid;
+  aid = agent_for_service(usock, "org.arl.unet.Services.PHYSICAL");
```

Comment:
Will this test fail against stacks without a PHYSICAL service agent?

Reply:
Yes, if there is no agent running in the master container providing PHYSICAL service, this will fail. Question is 

What is it that I can send a `ParameterReq` to in the master container for which I do not have to assume its presence?

All I need to know is whether the message was understood or not understood.

Reply:
I would guess `NodeInfo` could be one. Thoughts @mchitre ?

Reply:
I have a discussion about this with prof @mchitre. I will go with `NodeInfo`.

Reply:
Fixed now in 2148d38